planning_plugins:
  - ompl_interface/OMPLPlanner
# The order of the elements in the adapter corresponds to the order they are processed by the motion planning pipeline.
request_adapters:
  - default_planning_request_adapters/ResolveConstraintFrames
  - default_planning_request_adapters/ValidateWorkspaceBounds
#  - default_planning_request_adapters/CheckStartStateBounds
#  - default_planning_request_adapters/CheckStartStateCollision
response_adapters:
  - default_planning_response_adapters/AddTimeOptimalParameterization
  - default_planning_response_adapters/ValidateSolution
  - default_planning_response_adapters/DisplayMotionPath
planner_configs:
  RRTstar:
    type: geometric::RRTstar
    use_approximate_solution: true

ur_manipulator:
  default_planner_config: RRTstar
  planner_configs:
    - RRTstar
  projection_evaluator: joints(shoulder_pan_joint, shoulder_lift_joint, elbow_joint, wrist_1_joint, wrist_2_joint, wrist_3_joint)

start_state_max_bounds_error: 0.20
